### PR TITLE
Drop EOL version supports

### PR DESCRIPTION
**Pull request by Surgo**
Subject: Drop following EOL version supports

### Feature or Bugfix

- Feature

### Purpose

- Split a #29 PR

### Detail

Following versions are EOL

- 1.8
- 1.9
- 1.10

### Relates

- https://www.djangoproject.com/download/

### CODE REVIEW

**Comment by Surgo** [comment]
https://travis-ci.org/shimizukawa/django-redshift-backend/jobs/386139366

why cache wheel...

**Comment by shimizukawa** [comment]
Thanks! before merging this, I'm going to make a release in a couple of days.

**Comment by shimizukawa** [comment]
Now 0.8 has been shipped so it's time to merging this PR!

File: README.rst [hunk 1]
```diff
@@ -110,6 +107,7 @@ CHANGES
 0.9 (Unreleased)
 ----------------
 
+* Drop support for Django 1.8, 1.9 and 1.10
```
Comment (shimizukawa) [inline README.rst]: 🎉

**Pull request merged**